### PR TITLE
Update notebook with chatcompletion api

### PR DESCRIPTION
**Pull request by srbalakr**
## Purpose
Update notebook with chatcompletion api
* ...

## Does this introduce a breaking change?

```
[x ] Yes
[ ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[x ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
```

## What to Check
Verify that the following are valid
* ...

### CODE REVIEW

File: notebooks/chat-read-retrieve-read.ipynb [hunk 1]
```diff
@@ -159,7 +149,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.10.10"
+   "version": "3.11.4"
```
Comment (pamelafox) [inline notebooks/chat-read-retrieve-read.ipynb:152]: Not sure why this changed to 3.11, I guess you weren't in the 3.10 dev container when you edited it? Not a big deal, 3.11 should work too, just surprised by the change.

**Review by pamelafox — APPROVED** [review]
I find it hard to read Jupyter notebook diffs, but assuming that this works, it seems fine.

File: notebooks/chat-read-retrieve-read.ipynb [hunk 2]
```diff
@@ -20,10 +20,7 @@
     "AZURE_OPENAI_SERVICE = os.environ.get(\"AZURE_OPENAI_SERVICE\") or \"myopenai\"\n",
     "AZURE_OPENAI_GPT_DEPLOYMENT = os.environ.get(\"AZURE_OPENAI_GPT_DEPLOYMENT\") or \"davinci\"\n",
     "AZURE_OPENAI_CHATGPT_DEPLOYMENT = os.environ.get(\"AZURE_OPENAI_CHATGPT_DEPLOYMENT\") or \"chat\"\n",
-    "\n",
-    "KB_FIELDS_CONTENT = os.environ.get(\"KB_FIELDS_CONTENT\") or \"content\"\n",
```
Comment (chuwik) [inline notebooks/chat-read-retrieve-read.ipynb:24]: Aren't KB_FIELDS_CONTENT and KB_FIELDS_SOURCEPAGE used in line 125?

Comment (srbalakr) [reply notebooks/chat-read-retrieve-read.ipynb:24]: Thanks that was a miss . Updated.

**Review by chuwik — APPROVED** [review]

**Pull request merged**